jobs/build/pre-release/Jenkinsfile

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env groovy
+
+node {
+    checkout scm
+    def prerelease = load("../release/release.groovy")
+    def buildlib = prerelease.buildlib
+    def commonlib = prerelease.commonlib
+    def quay_url = "quay.io/openshift-release-dev/ocp-release-nightly"
+
+
+    // Expose properties for a parameterized build
+    properties(
+        [
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '')),
+            [
+                $class: 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    [
+                        name: 'FROM_RELEASE_TAG',
+                        description: 'Build tag to pull from (i.e. 4.1.0-0.nightly-2019-04-22-005054)',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: ""
+                    ],
+                    [
+                        name: 'NAME',
+                        description: 'Release name (i.e. 4.1.0-rc0)',
```

Comment:
Job should re-use the release tag unless this is specified. Should not need to specify.

Reply:
I am not sure what do you mean, release job also need to manually specified, so how do we reuse?

Reply:
say we are pre-releasing `registry.svc.ci.openshift.org/ocp/release:4.1.0-0.nightly-2019-07-24-213555`
then we are just creating `quay.io/openshift-release-dev/ocp-release-nightly:4.1.0-0.nightly-2019-07-24-213555`
there is no need to give it a new name, unless we want to for some reason

Reply:
done